docs/README.md

```diff
@@ -215,3 +215,14 @@ contribution](https://trino.io/development/process.html).
    Docker](https://www.youtube.com/watch?v=y58sb9bW2mA) gives you a starting
    point for setting up a test system on your laptop.
 
+## Release process
```

Comment:
"Release process" usually describes how to do a release (like: run this, publish that, etc.)
This is "Release cadence" or "Releases"